DSCResources/MSFT_xWebsite/MSFT_xWebsite.psm1

```diff
@@ -181,7 +201,29 @@ function Set-TargetResource
         $ServiceAutoStartProvider,
         
         [String]
-        $ApplicationType
+        $ApplicationType,
+
+        [String]
+        $LogPath,
+
+        [String[]]
+        [ValidateSet('Date','Time','ClientIP','UserName','SiteName','ComputerName','ServerIP','Method','UriStem','UriQuery','HttpStatus','Win32Status','BytesSent','BytesRecv','TimeTaken','ServerPort','UserAgent','Cookie','Referer','ProtocolVersion','Host','HttpSubStatus')]
```

Comment:
Can we change the order of the Type def and the Validate* Declaration - same with the remaining parameters. Cheers! :smile: